### ISSUE TITLE
graphics: Make (*Image).At and (*Image).Set available out of the main loop

### ISSUE THREAD

**Issue by hajimehoshi** [hajimehoshi/ebiten#790]
The current problem is that on mobiles GL drivers are not available outside the loop.

This would require implementing a software graphics driver.

**Comment by hajimehoshi** [comment]
Currently At before Run might freeze, which is a bad behavior.

**Comment by hajimehoshi** [comment]
https://github.com/golang/go/issues/25255 is the blocker.

**Comment by einthusan** [comment]
I don't think this is related exactly to this issue, but this article does talk about the performance of image.At in Go and how he managed to fix it (which isn't clear) but improved performance. Please take a quick look, short article.

https://medium.com/@marek.michalik/c-vs-rust-vs-go-performance-analysis-945ab749056c

**Comment by hajimehoshi** [comment]
Thanks but this is not related.

**Comment by hajimehoshi** [comment]
Now a graphics library is determined at Run, `(*ebiten.Image).At` is not available even if https://github.com/golang/go/issues/25255 is resolved.
